src/libs/network/manageNodeCall.ts

```
@@ -65,7 +66,7 @@ export async function manageNodeCall(content: NodeCall): Promise<RPCResponse> {
             response.response = await getPeerlist()
             break
         case "getPeerlistHash":
-            var peerlist = await getPeerlist()
+            let peerlist = await getPeerlist()
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)